definition-src/Game/LambdaHack/Definition/DefsInternal.hs

```diff
@@ -58,7 +60,7 @@ displayContentSymbol = id
 -- by accident (this is still possible via conversion functions,
 -- if one insists, so the abstraction is leaky, but that's fine).
 newtype ContentSymbol c = ContentSymbol Char
-  deriving (Show, Eq, Ord, Generic, Binary, NFData)
+  deriving (Show, Eq, Ord, Binary, NFData) -- Generic?
```

Comment:
`Generic` and most others are only needed for `TriggerItem`, so once the latter is removed, these instances can go.